src/components/Gradebook/gradebook.scss

```diff
@@ -98,4 +98,8 @@
 
 .mb-85 {
     margin-bottom: 85px;
+}
+
+.modal-dialog {
```

Comment:
We might want to change the specificity of this. This will change the max-widths of all modals that we can use in our application which isn't desirable. Unfortunately, from the quick look I took, it seems as though the modal component gets placed at the top of the DOM so I'm not exactly sure its possible without editing the Modal component in paragon. @matthugs do you have any thoughts on this?

Reply:
Discussed offline using the `parentSelector` prop of paragon's modal component to achieve this (using portals!) https://edx.github.io/paragon/?path=/story/modal--basic-usage